core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -2982,10 +2981,10 @@ final class Stream[+F[_], +O] private (private val free: FreeC[Nothing, O, Unit]
 
 object Stream extends StreamLowPriority {
   @inline private[fs2] def fromFreeC[F[_], O](free: FreeC[F, O, Unit]): Stream[F, O] =
-    new Stream(free.asInstanceOf[FreeC[Nothing, O, Unit]])
+    new Stream(free)
 
   /** Creates a pure stream that emits the supplied values. To convert to an effectful stream, use `covary`. */
-  def apply[F[x] >: Pure[x], O](os: O*): Stream[F, O] = emits(os)
+  def apply[O](os: O*): Stream[Pure, O] = emits(os)
```

Comment:
I think this one has the potential to impact lots of folks, though maybe not -- perhaps folks typically use `Stream(...).covary[F]`

Reply:
Note that, with a covariant `F`, we can deprecate the `covary` methods in favour of type-ascriptions, such as ```Scala
Stream(1): Stream[IO, Int]
```

Reply:
I'm 👎 on deprecating `covary`, it's much more handy than type ascriptions (especially because you only need to type in the effect type and the element type is inferred).

I rarely use `apply` with an explicit type though, so I'm okay with this particular change :)

Reply:
Yeah I much prefer `covary[F]` over type ascription as I don't have to repeat the output type -- and it's worse with pull where you have to specify both output and result types.

Reply:
I have undone the change to the `apply` method. In general, I am undoing any change to the public API that is not strictily necessary to introduce the covariance in this Pull Request.